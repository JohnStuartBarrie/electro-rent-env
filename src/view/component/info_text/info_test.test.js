import React from 'react';
import { shallow } from 'enzyme';
import InfoText from './index.js';

test('render a InfoText', () => {
  const wrapper = shallow(
      <InfoText>Hello Jest!</InfoText>
  );
  expect(wrapper).toMatchSnapshot();
});
