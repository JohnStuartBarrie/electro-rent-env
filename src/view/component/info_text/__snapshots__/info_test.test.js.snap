// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a InfoText 1`] = `
<div>
  Digging Electro Rent
  <p>
    Hello Jest!
  </p>
</div>
`;
